packages/@okta/vuepress-site/code/oie/index.md

```diff
@@ -0,0 +1,102 @@
+
+##### Android
+
+<!--* [Android Identity Engine SDK](https://github.com/okta/okta-idx-android)-->
+* [Android redirect sample app](https://github.com/okta/samples-android) &mdash; see [Browser Sign In](https://github.com/okta/samples-android/tree/master/browser-sign-in) for redirect configuration
+
+##### iOS
+
+* [Swift Identity Engine SDK](https://github.com/okta/okta-idx-swift)
+* [iOS embedded authentication with SDK sample app](https://github.com/okta/okta-idx-swift/tree/feat-dynamic-auth/Samples/EmbeddedAuthWithSDKs)
+* [iOS redirect sample app](https://github.com/okta/samples-ios) &mdash; see [Browser Sign In](https://github.com/okta/samples-ios/tree/master/browser-sign-in) for redirect configuration
+
+##### React Native
+
+* [React Native redirect sample app](https://github.com/okta/samples-js-react-native) &mdash; see [Browser Sign In](https://github.com/okta/samples-js-react-native/tree/master/browser-sign-in) for redirect configuration
+
+### Front End
+
+##### Angular
+
+* [Angular redirect sample app](https://github.com/okta/samples-js-angular) &mdash; see [Okta-Hosted Login](https://github.com/okta/samples-js-angular/tree/master/okta-hosted-login) for redirect configuration
+
+<!--
+#### JavaScript
+
+* [JavaScript Identity Engine SDK](https://github.com/okta/okta-idx-js)
+* [Sign-In Widget](https://github.com/okta/okta-signin-widget)
+-->
+
+##### React
+
+* [React redirect sample app](https://github.com/okta/samples-js-react) &mdash; see [Okta-Hosted Login](https://github.com/okta/samples-js-react/tree/master/okta-hosted-login) for redirect configuration
+
+##### Vue
+
+* [Vue redirect sample app](https://github.com/okta/samples-js-vue) &mdash; see [Okta-Hosted Login](https://github.com/okta/samples-js-vue/tree/master/okta-hosted-login) for redirect configuration
+
+### Back End
+
+##### .NET
+
+* [.NET Identity Engine SDK](https://github.com/okta/okta-idx-dotnet)
+* [ASP.NET MVC embedded authentication with SDK sample app](https://github.com/okta/okta-idx-dotnet/tree/master/samples/samples-aspnet/embedded-auth-with-sdk)
+
+* [ASP.NET MVC redirect sample app](https://github.com/okta/samples-aspnet) &mdash;  see [Okta-Hosted Login](https://github.com/okta/samples-aspnet/tree/master/okta-hosted-login) for redirect configuration
+* [ASP.NET Web Forms redirect sample app](https://github.com/okta/samples-aspnet-webforms) &mdash;  see [Okta-Hosted Login](https://github.com/okta/samples-aspnet-webforms/tree/master/okta-hosted-login) for redirect configuration
+* [Blazor redirect sample app](https://github.com/okta/samples-blazor) &mdash;  see [Blazor Server-Side Okta-Hosted Login](https://github.com/okta/samples-blazor/tree/master/server-side/okta-hosted-login) for redirect configuration
+* [ASP.NET Core 2.x and 3.x redirect sample app](https://github.com/okta/samples-aspnetcore) &mdash;  see [Okta-Hosted Login](https://github.com/okta/samples-aspnetcore/tree/master/samples-aspnetcore-2x/okta-hosted-login) for redirect configuration
```

Comment:
this should only reference the 3.x asp.net core - 2.x is no longer supported. Also change the link to the 3x directory for this one.

Reply:
thanks!